root/release/edit/tracklist.tt

```diff
@@ -432,8 +432,12 @@
         [%~ l('Tracks are required. If you don’t know them, but do know some info (such as the format or number of media), tick the “I don’t know the tracklist for this medium” checkbox above and enter the available info. If you have no information about this release’s media, remove all media.') ~%]
       </p>
 
-      <p class="error field-error" style="padding: 1em" data-bind="showErrorWhenTabIsSwitched: needsTrackInfo">
-        [%~ l('You must enter a title and select an artist for every track.') ~%]
+      <p class="error field-error" style="padding: 1em" data-bind="showErrorWhenTabIsSwitched: needsTrackTitles">
+        [%~ l('You must enter a title for every track (if the track is untitled, enter “[untitled]”).') ~%]
+      </p>
+
+      <p class="error field-error" style="padding: 1em" data-bind="showErrorWhenTabIsSwitched: needsTrackArtists">
+        [%~ l('You must select an artist for every track (the background color of every artist field should be green). If you’ve entered an artist but the background is white, click the magnifying glass to match the name to a MusicBrainz artist. If the artist doesn’t exist yet, you can create it from the search results.') ~%]
```

Comment:
1. It is “white” only using the current style, but it can be styled using external tools already, and after MBS-10444 in the future. “uncolored” seems to be a more long-standing drop-in replacement.
2. It can be more specific about the location to “Add a new artist”.

```suggestion
        [%~ l('You must select an artist for every track (the background color of every artist field should be green). If you’ve entered an artist but the background remains uncolored, click the magnifying glass to match the name to a MusicBrainz artist. If the artist doesn’t exist yet, you can create it from below the search results in the drop-down.') ~%]
```

Reply:
The uncolored suggestions sounds good to me.

For the second suggestion, maybe we might as well specify the name of the menu item? We could use a parameter to ensure the string is identical: `you can create it by selecting “{add_a_new_artist}” from the bottom of the search results drop-down.', {add_a_new_artist => l('Add a new artist')}`.

Reply:
Will that work? I am having issues with a similar idea in https://github.com/metabrainz/musicbrainz-server/pull/2778

Reply:
Can I just say "is not green"? "uncolored" seems like a more complicated word than we need (you can kinda guess what it means as a person with bad English, sure, but you might never have heard it before).

Reply:
Hmm. After sleeping on this for a couple days, uncolored is probably fine anyway, so I'll use that :)